lib/src/sai_redis_generic_remove.cpp

```diff
@@ -118,10 +118,10 @@ sai_status_t internal_redis_bulk_generic_remove(
         }
 
         /*
-         * Capital 'C' stands for bulk CREATE operation.
+         * Capital 'R' stands for bulk CREATE operation.
          */
 
-        recordLine("C|" + str_object_type + joined);
+        recordLine("R|" + str_object_type + joined);
```

Comment:
>R [](start = 20, length = 1)

Is there a wiki page to summarize all the special char used in recording? If not, please create one.

Alternatively, how about centralized the definition in one header file and use constants or macros in code?

Reply:
Could we set such a rule that,  small letter means request, capital letter is for response? 
 Ex:  
```
g/G(get request/Get response), 
c/C (create request/create reponse).  
r/R(remove request/Remove response).  
s/S(set request/Set reponse)
b/B(buld create/Bulk create response)
e/E(bulk remove request/Bulk remove response)
m/M(bulk set request/Buld set reponse)
```

Reply:
actually m/M should be removed since it's not SAI api call, its's request for counters subscription now, and recording should log only sai call's
also single 'b' is not good enough since you can have bulk remove, bulk create and so on

Reply:
Thanks @jipanyang ! I still suggest to maintain it as any of the below options:
1. code comment or Github wiki page and referred by code comment.
2. centralized the definition in one header file and use constants/macros/enums in code?

Nowadays it is hard to find, and easy to conflict.